loopchain/blockchain/vote.py

```diff
@@ -111,73 +107,33 @@ def get_voters(self):
     def get_result(self, block_hash, voting_ratio):
         return self.get_result_detail(block_hash, voting_ratio).result
 
-    def get_result_detail(self, block_hash, voting_ratio) -> VoteResult:
+    def get_result_detail(self, target_hash, voting_ratio) -> VoteResult:
         """
 
-        :param block_hash:
+        :param target_hash:
         :param voting_ratio:
-        :return: result(str),
-        agree_vote_group_count, total_vote_group_count, total_group_count,
-        agree_vote_peer_count, total_peer_count, voting_ratio
+        :return: result(str), total_vote_count, agree_vote_peer_count, total_peer_count, voting_ratio
         """
 
-        if self.__target_hash != block_hash:
-            return None, 0, 0, 0, 0, 0, 0, 0
-
-        total_group_count = len(self.__votes)
-        total_peer_count = sum([len(self.__votes[group_id]) for group_id in list(self.__votes.keys())])
-        agree_vote_group_count = 0
-        total_vote_count = 0
-        total_vote_group_count = 0
-        agree_vote_peer_count = 0
-        result = None
+        if self.__target_hash != target_hash:
+            return None, -1, -1, -1
```

Comment:
How about returning it like below to clarify codes?
```python
return VoteResult(
    result = None, 
    total_vote_count = -1, 
    arggree_vote_peer_count=-1, 
    total_peer_count=-1, 
    voting_ratio=-1
)
```
```

Reply:
👍